### DISCUSSION TITLE
Any idea how carbon handle exceptions/errors? Support try/catch?

### DISCUSSION THREAD

**Discussion by sirlordt** [carbon-language/carbon-lang#1931] [Language design]
Hello. Any progress how handle exceptions or errors in carbon?.

Carbon will be support to try/catch?

The explorer not had any example about this issue.

**Comment by geoffromer** [comment]
We've documented some principles [here](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/principles/error_handling.md), but we don't have a concrete design yet.

(I'm converting this issue to a discussion because there's no action we can take to resolve it)
